way_to_home/tests/unittests/middlewares/testloginrequired.py

```diff
@@ -0,0 +1,50 @@
+"""This module provides tests for custom LoginRequired middleware."""
+
+from django.test import TestCase, Client
+
+from custom_user.models import CustomUser
+from middlewares.login_required import GUESTS_PATHS
+
+
+class LoginRequiredTestCase(TestCase):
+    """TestCase for providing LoginRequired middleware testing."""
+
+    def setUp(self):
+        """Provide preparation before testing middleware."""
+        user = CustomUser(id=100, email='testuser@mail.com', is_active=True)
+        user.set_password('testpassword')
```

Comment:
Password like this doesn't pass Password validation

Reply:
I think it is not so important :)
But I changed it.